gradle.properties

```
@@ -14,14 +14,20 @@ profile=local
 ## Local (default)
 checks.local.sharded=true
 checks.local.configFiles=file://@ROOTDIR@/config/configuration.json
-checks.local.input=file://@BUILDDIR@/example/data/atlas/
+checks.local.input=file://@BUILDDIR@/../atlas/
+#checks.local.input=hdfs://10.135.134.89:9000/tsmock/atlas/
```

Guessing you want to take this out of the pr
```suggestion
checks.local.input=file://@BUILDDIR@/example/data/atlas/
```